root/static/scripts/common/MB.js

```diff
@@ -39,4 +39,4 @@ const MB: {[string]: *} = {
 
 global.MB = MB;
 
-module.exports = MB;
+export default MB;
```

Comment:
There's a `require` in root/static/scripts/common/MB/edit_search.js still which doesn't have `.default` added.